engine/lib/navigation.php

```diff
@@ -126,7 +130,7 @@ function elgg_register_menu_item($menu_name, $menu_item) {
  * @since 1.8.0
  */
 function elgg_unregister_menu_item($menu_name, $item_name) {
-	global $CONFIG;
+	$CONFIG = _elgg_services()->config->getStorageObject();
```

Comment:
why not use `$menus = elgg_get_config('menus');` like you do in the other functions in this file?